packages/consumption/src/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.ts

```diff
@@ -361,7 +363,8 @@ export class ReadAttributeRequestItemProcessor extends GenericRequestItemProcess
                 id: responseItem.attributeId,
                 content: responseItem.attribute,
                 peer: requestInfo.peer,
-                requestReference: requestInfo.id
+                requestReference: requestInfo.id,
+                thirdPartyAddress: responseItem.thirdPartyAddress
             });
         }
 
```

Comment:
In the following, the succession of ThirdPartyRelationshipAttributes comes into play. When calling the function `succeedThirdPartyOwnedRelationshipAttribute`, the `thirdPartyAddress` needs to be provided within the `successorParams` as well.